src/debug.c

```diff
@@ -551,11 +551,9 @@ NULL
         emptyDb(-1,EMPTYDB_NO_FLAGS,NULL);
         protectClient(c);
         int ret = loadAppendOnlyFile(server.aof_filename);
+        if (ret != AOF_OK && ret != AOF_EMPTY)
+            exit(1);
```

Comment:
Why did we choose exit here. A DEBUG command can make the server exit, is a bit odd.

I encountered this issue:
start a server with AOF off
debug loadaof
the server exit

the reason is that, in loadAppendOnlyFiles, the ret is AOF_NOT_EXIST, so it do a exit(1)

i think we should also ignore AOF_NOT_EXIST in this case?
or actually we need to return an error, print the log, instead of exit

Reply:
this command is only meant to be used by the test suite, and only by tests that generated an AOF file first.
i don't think it should ignore the AOF_NOT_EXIST, but i'm willing to trade this exit with an error reply (considering that the caller is likely to catch this error and die)